changelog/8509.improvement.rst

```diff
@@ -1,5 +1,5 @@
-Fixed issue where `TestCase.setUpClass` is not called when a test has `/` in its name since pytest 6.2.0.
+Fixed issue where :method:`unittest.TestCase.setUpClass` is not called when a test has `/` in its name since pytest 6.2.0.
```

Comment:
My bad, this should be 

```suggestion
Fixed issue where :meth:`unittest.TestCase.setUpClass` is not called when a test has `/` in its name since pytest 6.2.0.
```

Reply:
Cool, thank you so much for the pointers! I made another commit. Let me know when you have resolved changelog/8761.feature.rst and changelog/8645.improvement.rst and I'll make those changes ASAP